packages/rs-drive-abci/src/execution/storage/fetch_execution_state/v0/mod.rs

```diff
@@ -0,0 +1,24 @@
+use crate::error::Error;
+use crate::platform_types::platform_state::PlatformState;
+use dpp::serialization::PlatformDeserializableFromVersionedStructure;
+use dpp::version::PlatformVersion;
+use drive::drive::Drive;
+use drive::query::TransactionArg;
+
+pub(super) fn fetch_execution_state_v0(
+    drive: &Drive,
+    transaction: TransactionArg,
+    platform_version: &PlatformVersion,
+) -> Result<Option<PlatformState>, Error> {
+    let maybe_bytes = drive
+        .fetch_execution_state_bytes(transaction, platform_version)
+        .map_err(Error::Drive)?;
```

Comment:
you should just .map here, instead of the next few lines.

Reply:
Good point! Thank you